src/__tests__/test-service.ts

```diff
@@ -303,11 +319,9 @@ export class TestTigrisService {
 				call.request.getOptions().getLimit() == 1 &&
 				filter["id"] == 1
 			) {
-				// base64 of {"id":1,"title":"A Passage to India","author":"E.M. Forster","tags":["Novel","India"]}
+				// base64 of book id "1"
 				call.write(
-					new ReadResponse().setData(
-						"eyJpZCI6MSwidGl0bGUiOiJBIFBhc3NhZ2UgdG8gSW5kaWEiLCJhdXRob3IiOiJFLk0uIEZvcnN0ZXIiLCJ0YWdzIjpbIk5vdmVsIiwiSW5kaWEiXX0="
-					)
```

Comment:
moved all these responses to a `BOOKS_B64_BY_ID` map, no changes to this method's behavior